docs/topics/modules/glossary.adoc

```diff
@@ -37,10 +37,6 @@ area:: Use areas to organize work items to distinguish between different types o
 backlog:: A backlog is a list of work items that have not been triaged and assigned to a specific iteration.
 // endterm
 
-// term: e8d54bf3-f89e-46e5-86f7-4af6475863b0, en_EN
```

Comment:
It's the definition for "bug", which doesn't have any special meaning aside from the very common general meaning for IT. Happy to add it back, though.

Reply:
@Preeticp I think you added this initially. What's the reasoning for having a "bug" definition?

Reply:
This is part of the SDD work item type (WIT) hierarchy, and would have an info tip associated with it. I can check with the Planner team if we are ok with removing it, since it is easily understood. Importantly, currently, in SDD, Bugs come at two levels in the WIT hierarchy with different emphasis, I was told. One was to be removed, will check on that too and update.

Reply:
I checked with the Planner team today during stand up, they felt we should retain the Bug definition for infotips as SDD has many unique WITs and it is better for clarity and consistency to retain it.

Reply:
I've added the bug definition back in the next PR anyway so we should be all good (unless we decide later to remove it for any reason).